src/LCDC.md

```diff
@@ -110,6 +110,8 @@ When Bit 0 is cleared, the background and window lose their priority -
 the sprites will be always displayed on top of background and window,
 independently of the priority flags in OAM and BG Map attributes.
 
+[See background and window to obj priority](Tile_Maps.md#bg-to-obj-priority-in-cgb-mode)
```

Comment:
It's not very clear to me why the pointer is here. E.g.: how should the reader decide to click here? What kind of additional information do we need to convery that requires adding a pointer?
E.g. *the behaviour needs to be expanded because of an edge case that...*

Reply:
Good point, I hope its clearer now